--- origsrc/scratchpad-0.3.0/scratchpad/Makefile.am	2006-09-21 13:34:26.000000000 -0500
+++ src/scratchpad-0.3.0/scratchpad/Makefile.am	2008-07-29 23:47:00.062500000 -0500
@@ -2,7 +2,7 @@
 
 INCLUDES = \
 	$(SCRATCHPAD_CFLAGS) 	\
-	-I../libguniqueapp		\
+	-I$(top_srcdir)/libguniqueapp	\
 	$(NULL)
 	
 BUILT_SOURCES = scratchpad.stamp
@@ -45,8 +45,8 @@
 	touch $@
 	
 scratchpad_LDADD = \
-	$(SCRATCHPAD_LIBS) 					\
 	../libguniqueapp/libguniqueapp.la 	\
+	$(SCRATCHPAD_LIBS) 					\
 	$(NULL)
 	
 EXTRA_DIST = scratchpad.stamp
